resolveImports.mjs

```diff
@@ -11,8 +23,14 @@ async function exists(module) {
   }
 }
 
+/**
+ * Recursively parses and resolves a module's imports.
+ * @param {string} module The path to the module.
+ * @param {boolean} [root=true] Whether the module is the root module.
```

Comment:
```suggestion
 * @param {boolean} [root] Whether the module is the root module.
```

TypeScript flavour of JSDoc doesn't do anything with default values, and to be DRY I prefer to have one source of truth what the default value is (the actual code). Default values often can't be communicated inline the trad JSDoc way, because there can be conditional logic to explain about what the default is. Also, if the default value is not set by the function parameter default syntax, and is instead set inside the function by some kind of logic, it's kind of incorrect to declare a parameter default value in the JSDoc. IMO the best way to document default values or behaviors, is to have a final sentence in the parameter JSDoc description, starting with `Defaults to ` and then a description of what the default value or logic is.

Reply:
Oh that's good, I wasn't a fan of the duplication either.